tests/test_affinity.py

```diff
@@ -7,7 +7,7 @@
 from shapely.wkt import loads as load_wkt
 from shapely.geometry import Point
 
-from . import unittest
+import unittest
```

Comment:
Here and below, move std lib imports to top and at least put `unittest` into sorted order, then separate each group of std lib, 3rd party, shapely imports into separate blocks separated by new lines.

Reply:
this shall be automatically fixed by pre-commit once https://github.com/shapely/shapely/pull/1681 is merged.